runes/rune_test.go

```diff
@@ -0,0 +1,15 @@
+package runes_test
+
+import (
+	"testing"
+
+	"github.com/sygaldry/sygaldry-core/runes"
+)
+
+// TestRuneRun tests rune.Run
+func TestRuneRun(t *testing.T) {
+	myFirstRune := runes.NewRune("hello-world", []string{"TEST=test"})
+	if error := myFirstRune.Run(); error != nil {
+		t.Error("Expected success, got ", error)
```

Comment:
I was getting “repository name must be canonical” when running it, so I replaced the image name in the run with “docker.io/library/node:7.8.0-alpine” and then got this error “Error response from daemon: page not found”. looks like it’s actually trying to pull a docker image when I run the tests

Reply:
Fixed!